code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -685,3 +762,148 @@
+/datum/action/ability/activable/xeno/pounce/lunge/trigger_pounce_effect(mob/living/living_target)
+	playsound(get_turf(living_target), 'sound/voice/alien/roar4.ogg', 25, TRUE)
+	var/mob/living/carbon/xenomorph/xeno_owner = owner
+	xeno_owner.UnarmedAttack(living_target)
+	step_away(living_target, xeno_owner, 1, 3)
+	xeno_owner.face_atom(living_target)
+	living_target.Stun(XENO_POUNCE_STUN_DURATION) // no disarm but standing stun to make it feel like you got bodyslammed by a xeno
+
+// ***************************************
+// *********** Phase Out
+// ***************************************
+
+/datum/action/ability/xeno_action/stealth/phaseout
+	name = "Phase Out"
+	action_icon_state = "hunter_invisibility"
+	action_icon = 'icons/Xeno/actions/hunter.dmi'
+	desc = "Become fully invisible for 10 seconds, or until damaged. Attacking does not break invisibility."
+	ability_cost = 10
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_TOGGLE_PHASEOUT,
+	)
+	cooldown_duration = 10 SECONDS
+	stealth_alpha_multiplier = 1
+	stealth_duration = 10 SECONDS
+	disable_on_signals = list(
+		COMSIG_LIVING_IGNITED,
+		COMSIG_LIVING_ADD_VENTCRAWL,
+	)
+	disable_on_obj_slash = FALSE
+	disable_on_mob_slash = FALSE
+	sneak_attack_armor_pen = 30
+	pounce_break_stealth = FALSE
+	pounce_hit_break_stealth = TRUE
+
+// i just realized, we need to disable processing, this will inherently just amke handle_stealth not do anything
+///Updates or cancels stealth
+/datum/action/ability/xeno_action/stealth/phaseout/handle_stealth()
+	var/mob/living/carbon/xenomorph/xenoowner = owner
+	xenoowner.alpha = HUNTER_STEALTH_STILL_ALPHA * stealth_alpha_multiplier // instant full stealth regardless of movement.
+
+// ***************************************
+// *********** Death Mark
+// ***************************************
+/datum/action/ability/activable/xeno/hunter_mark/assassin
+	name = "Death Mark"
+	action_icon_state = "death_mark"
+	action_icon = 'icons/Xeno/actions/hunter.dmi'
+	desc = "Psionically disturb a creature for 15 seconds, allowing you to deal double sneak attack damage and stun from it. They will know you are coming for them."
+	ability_cost = 50
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_HUNTER_MARK,
+	)
+	cooldown_duration = 30 SECONDS
+	require_los = FALSE
+	timeout = 15 SECONDS
+	warntarget = TRUE
+	chargeup = 2 SECONDS
+
+// ***************************************
+// *********** Displacement
+// ***************************************
+
+/datum/action/ability/xeno_action/displacement
+	name = "Displacement"
+	action_icon_state = "hunter_invisibility"
+	action_icon = 'icons/Xeno/actions/hunter.dmi'
+	desc = "Physically disappear, become incorporeal until you decide to reappear somewhere else, reappearing on lighted areas will disorient you and flicker the lights."
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOMORPH_HUNTER_DISPLACEMENT,
+	)
+	use_state_flags = ABILITY_USE_SOLIDOBJECT
```

Comment:
This whole ability is extremely wack. Its infinite invulnerability, so you can safely scout anywhere, and unless marines get every last tile of resin, they can just pop up basically anywhere.

Its also visually identical to actual stealth, so marines have zero way to know what xenos can even be shot at.

Reply:
Guess i'll make it fully invisible, lights also slam the assassin pretty bad with slowdown and stagger so they actually need to rematerialize in a non lit turf, the flicker warns nearby marines in lighted rooms too. And materializing in light sends a massive red warning to all marines in vicinity letting them know.

Reply:
now i made it so you need to be in dark to pop into it, and its fully invisible now so it wont be a decoy of sorts and confusing, it flickers lights at a further range and less obstructively, really brief flicker.